proto/viam/app/v1/billing.proto

```diff
@@ -105,10 +106,61 @@ message GetItemizedInvoiceResponse {
   Invoice invoice = 1;
 }
 
+// Begin Suggestion 1:
+message GetPaymentMethodCardRequest {
+  string org_id = 1;
+}
+
+message GetPaymentMethodCardResponse {
```

Comment:
We probably don't need GetPaymentMethodResponse and GetPaymentMethodCardResponse as separate rpc calls. The GetPaymentMethodResponse with a type parameter and maybe an optional object representing a card inside of that as you have done below looks good!

That being said, we may want something a bit more verbose than GetPaymentMethod as an rpc call, but something that gets all the stripe <> org information. Apart from the payment details, the rest of the stripe objects are defined on the org model but I think we will want to return billing_email in the same request here. 

A nice way of thinking about it is where do the rpc calls hit, so if we require looking at org table to get payment info (we need to get the stripe customer id to then get the payment info from stripe), pulling the billing email there also makes sense.